### DISCUSSION TITLE
Relations in Filament Select

### DISCUSSION THREAD

**Discussion by vomojola** [staudenmeir/laravel-adjacency-list#244] [Q&A]
Please, hos can I use the built in relations in Filament select component? 

I tried to use the decesndantsAndSelf relationship to generate select list in FilamentPhp and this is the error:

Filament\Support\Services\RelationshipJoiner::prepareQueryForNoConstraints(): Argument #1 ($relationship) must be of type Illuminate\Database\Eloquent\Relations\Relation, null given

Thank you.

**Comment by staudenmeir** [comment]
Hi @vomojola,
I haven't used the package with Filament. Can you create a repository for me to reproduce the issue?
